modules/calypsoify/style.scss

```diff
@@ -1,4 +1,5 @@
 @import 'colors';
+@import '../../node_modules/@automattic/calypso-color-schemes/dist/calypso-color-schemes.css';
```

Comment:
Note that you might need to remove or modify similar import in `_colors.scss`:

https://github.com/Automattic/jetpack/blob/66fdea2069b7dca13992a64512fdc3a68f727b72/modules/calypsoify/_colors.scss#L1

Reply:
good catch, thanks 👍

Reply:
I think that file would go away as soon as we moved everything over to css vars.